ai_edge_torch/generative/examples/gemma/gemma.py

```diff
@@ -36,16 +37,73 @@
     attn_value_proj="model.layers.{}.self_attn.v_proj",
     attn_output_proj="model.layers.{}.self_attn.o_proj",
     pre_attn_norm="model.layers.{}.input_layernorm",
-    pre_ff_norm="model.layers.{}.post_attention_layernorm",
+    post_attn_norm="model.layers.{}.post_attention_layernorm",
     embedding="model.embed_tokens",
     final_norm="model.norm",
     lm_head=None,
 )
 
+TENSOR_NAMES_GEMMA2 = loading_utils.ModelLoader.TensorNames(
```

Comment:
Can we keep gemma2 related code in a new file? this might be easier for future development.

Reply:
+1 on separating Gemma2 to another file. 

I know that would not be DRY but a little bit of repetition should be ok if it make the code easier to follow. At the end of the day, most of the common blocks are moved out to the layers directory.

Reply:
Moved Gemma2 to a new file. PTAL.